Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -710,6 +714,22 @@ ALLOW_DEPRECATED_DECLARATIONS_END
     bool inlinePredictionsEnabled() const { return m_inlinePredictionsEnabled; }
 #endif
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+    void willBeginTextReplacementSession(const WTF::UUID&, CompletionHandler<void(const Vector<WebKit::WebUnifiedTextReplacementContextData>&)>&&);
```

Comment:
Nit - you don't need the explicit `WebKit::` namespacing here, because we're already in the WebKit namespace.